x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/get_error_count_chart_preview.ts

```diff
@@ -66,7 +63,7 @@ export async function getTransactionErrorCountChartPreview({
     bool: {
       filter: [
         ...termFilterQuery,
-        ...getParsedFilterQuery(kqlFilter),
+        ...getParsedFilterQuery(searchConfiguration),
```

Comment:
If you do this:

```suggestion
        ...getParsedFilterQuery(searchConfiguration?.query?.query),
```

You can use the existing `getParsedFilterQuery` instead of duplicating

Reply:
updated in 627420f13d6c4587758e54ee70f57de143dad846